tests/integration/targets/ini_file/tasks/tests/02-values.yml

```diff
@@ -0,0 +1,820 @@
+    state: present
+    exclusive: false
+  register: result5
+
+- name: test-values 5 - read content from output file
+  slurp:
+    src: "{{ output_file }}"
+  register: output_content
+
+- name: test-values 5 - set expected content and get current ini file content
+  set_fact:
+    expected5: |
+
+      [drinks]
+      fav = cocktail
+      fav = lemonade
+    content5: "{{ output_content.content | b64decode }}"
+
+- name: test-values 5 - Verify content of ini file is as expected and ini_file 'changed' is true
+  assert:
+    that:
+      - result5 is changed
+      - result5.msg == 'option added'
+      - content5 == expected5
+
+
+- name: "test-values 6 - ensure 'fav=lemonade' and 'fav=cocktail' is 'present' in section '[drinks]' and check for idempotency"
+  ini_file:
+    path: "{{ output_file }}"
+    section: drinks
+    option: fav
+    values:
+      - lemonade
+      - cocktail
+    state: present
+  register: result6
+
+- name: test-values 6 - Ensure unchanged
+  assert:
+    that:
+      - result6 is not changed
+      - result6.msg == 'OK'
+
+
+- name: "test-values 7 - ensure 'fav=cocktail' and 'fav=lemonade' (list reverse order) is 'present' in section '[drinks]' and check for idempotency"
+  ini_file:
+    path: "{{ output_file }}"
+    section: drinks
+    option: fav
+    values:
+      - cocktail
+      - lemonade
+    state: present
+  register: result7
+
+- name: test-values 7 - Ensure unchanged
+  assert:
+    that:
+      - result7 is not changed
+      - result7.msg == 'OK'
+
+
+- name: "test-values 8 - add option 'fav=lemonade' in section '[drinks]' again and ensure idempotency"
+  ini_file:
+    path: "{{ output_file }}"
+    section: drinks
+    option: fav
+    values:
+      - lemonade
+    state: present
+    exclusive: false
+  register: result8
+
+- name: test-values 8 - Ensure unchanged
+  assert:
+    that:
+      - result8 is not changed
+      - result8.msg == 'OK'
+
+
+- name: "test-values 9 - ensure only 'fav=lemonade' is 'present' in section '[drinks]' in specified file"
+  ini_file:
+    path: "{{ output_file }}"
+    section: drinks
+    option: fav
+    values:
+      - lemonade
+    state: present
+  register: result9
+
+- name: test-values 9 - read content from output file
+  slurp:
+    src: "{{ output_file }}"
+  register: output_content
+
+- name: test-values 9 - set expected content and get current ini file content
+  set_fact:
+    expected9: |
+
+      [drinks]
+      fav = lemonade
+    content9: "{{ output_content.content | b64decode }}"
+
+- name: test-values 9 - Verify content of ini file is as expected and ini_file 'changed' is true
+  assert:
+    that:
+      - result9 is changed
+      - result9.msg == 'option changed'
+      - content9 == expected9
+
+
+- name: "test-values 10 - remove non-existant 'fav=cocktail' from section '[drinks]' in specified file"
+  ini_file:
+    path: "{{ output_file }}"
+    section: drinks
+    option: fav
+    values:
+      - cocktail
+    state: absent
```

Comment:
prior to this PR (and with `value=cocktail` instead), this would have removed the `fav` option entirely. It seems this is no more the case.

Reply:
I changed the test. It now removes the entire `fav` options, because `exclusive` is `true` by default.